uniffi_bindgen/src/backend/types.rs

```diff
@@ -16,6 +16,10 @@ pub trait CodeType: Debug {
     /// method signatures and property declarations.
     fn type_label(&self) -> String;
 
+    fn protocol_label(&self) -> String {
```

Comment:
"protocol" doesn't mean anything to other bindings IIUC, and there's a chance that it means something *different* in some external languages. Do you think it might be possible to keep this abstraction in the Kotlin bindings? It seems like that's where some types get an overridden value, so if we can encapsulate the defaults there too it seems like it might be an improvement.

Reply:
I can try, maybe I can make it work with some filter, but it depends on what info I have available there.

Reply:
You might be able to add it to the `KotlinCodeOracle` too? That can get quite confusing, feel free to ping us if you get stuck.

Reply:
Done in https://github.com/mozilla/uniffi-rs/pull/1782/commits/451480c7ac8fcbda5b0d0cf83305ebcb351fa447